### PR TITLE
First ML project.yml

### PR DESCRIPTION
**Pull request by snehit31**
Hi This is an update for all ML,data scientist enthusiast.

### CODE REVIEW

File: _data/projects/ML [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: ML
+desc: Project on Linear Regression
+site: datascience.com/highest-rated-ml-projects-on-github-694486293
```
Comment (shiftkey) [inline _data/projects/ML:3]: Is this link correct? For me it redirects to `https://www.oracle.com/corporate/acquisitions/datascience/` which doesn't seem relevant.

File: _data/projects/ML [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- Regression
+- ML
+- Data Science
+- AI
+upforgrabs:
+  name: Sexiest Job 21st Century
+  link: https://github.com/snehit31/up-for-grabs.net
```
Comment (shiftkey) [inline _data/projects/ML:12]: This is your fork of the Up for Grabs repository.

File: _data/projects/ML [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+desc: Project on Linear Regression
+site: datascience.com/highest-rated-ml-projects-on-github-694486293
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- Regression
+- ML
+- Data Science
+- AI
+upforgrabs:
+  name: Sexiest Job 21st Century
```
Comment (shiftkey) [inline _data/projects/ML:11]: I hope this is not a real label you plan to use

**Review by shiftkey — CHANGES_REQUESTED** [review]

**Comment by shiftkey** [comment]
@snehit31 you have currently opened six pull requests against the project in the last 48 hours - #1593 #1594 #1595 #1596 #1608 #1609 - and the details in this one make me question whether you are looking at the review feedback and want to actually list some projects, or are instead just opening pull requests to try and achieve some goal.

I don't want to ban anyone from this project, but I cannot tolerate this behaviour any more.

**This is your first official warning. Please stop opening new pull requests, and either address or close the pull requests you have previously opened before opening any more. If you continue to open pull requests I will block you from this project.**

@snehit31 please confirm you understand what I am saying and why I am saying it.

**Comment by snehit31** [comment]
Yes sir,I understood your concern and apologize for this behaviour.Actually it was my first repository on which I was working and I was just experimenting to understand.I would'nt be making any more pull request.

**Pull request closed without merging**